trace/trace.go

```diff
@@ -149,8 +150,15 @@ func (tr *tracePlugin) AfterExecution(ctx context.Context, request json.RawMessa
 		}
 	}
 
-	// TODO: Serialize response properly
-	tr.RootSpan.SetTag(constants.AwsLambdaInvocationResponse, response)
+	enableResponseData := true
```

Comment:
I think we can directly use `config.TraceResponseDisabled ` and `config.TraceRequestDisabled ` variables instead of creating new `enableResponseData` and `enableRequestData` variables.